compiler/rustc_middle/src/mir/spanview.rs

```diff
@@ -16,7 +16,8 @@ const ANNOTATION_RIGHT_BRACKET: char = '\u{2989}'; // Unicode `Z NOTATION LEFT B
 const NEW_LINE_SPAN: &str = "</span>\n<span class=\"line\">";
 const HEADER: &str = r#"<!DOCTYPE html>
 <html>
```

Comment:
while we're at it, could you also add `lang="en"` to the html tag?

Reply:
Done.